active/tab/tabcontrol.md

```diff
@@ -0,0 +1,252 @@
+example code with each description. The general format is: 
+  feature explanation,
+  example code
+  feature explanation,
+  example code
+  etc.-->
+
+<!-- As an example of this section, see the Examples section for the PasswordBox control 
+(https://docs.microsoft.com/windows/uwp/design/controls-and-patterns/password-box#examples). -->
+
+## To replicate the behavior of Microsoft Edge
+
+``` xml
+<TabControl TabWidthBehavior="Equal"
+            CanCloseTabs="True"
+            CloseButtonOverlay="OnHover"
+            CanDragItems="True"
+            CanReorderItems="True"
+            TabDraggedOutside="OpenTabInNewWindow">
+    <TabControl.TabActionContent>
+        <Button Style="{ThemeResource AddNewTabButtonStyle}" Click="NewTab_Click" />
+    </TabControl.TabActionContent>
+    ...
+</TabControl>
+```
+
+## Put tabs in the titlebar area
+
+![A sample app that shows the Tab control can extend into the titlebar area](images/tab-extend-to-title.png)
+
+This sample demonstrates how to extend the TabControl into the title bar area and also specify a portion of the UI as the draggable region. Per the [titlebar customization guidelines](https://docs.microsoft.com/en-us/windows/uwp/design/shell/title-bar), we must set a [specific drag region](https://docs.microsoft.com/en-us/windows/uwp/design/shell/title-bar#draggable-regions). If we don't specify the drag region, the entire titlebar area remains draggable (meaning input won't be routed to the tabs, making the tabs unclickable).
+
+``` xml
+<Page>
+    <Grid>
+        <TabControl>
+            <TabItem Icon="Home" Header="Home" IsCloseable="False" />
+            <TabItem Icon="Document" Header="Document 1" />
+            <TabItem Icon="Document" Header="Document 2" />
+            <TabItem Icon="Document" Header="Document 3" />
+        </TabControl>
+
+        <Grid x:Name="CustomDragRegion" Width="200" Height="40" HorizontalAlignment="Right" VerticalAlignment="Top" />
+    </Grid>
+</Page>
+```
+
+``` csharp
+// Customize the titlebar area using the guidance from here: https://docs.microsoft.com/en-us/windows/uwp/design/shell/title-bar
+public MainPage()
+{
+    this.InitializeComponent();
+
+    var coreTitleBar = CoreApplication.GetCurrentView().TitleBar;
+    coreTitleBar.ExtendViewIntoTitleBar = true;
+
+    Window.Current.SetTitleBar(CustomDragRegion);
+}
+```
+
+## Create a new window when tearing out tabs
+
+![An example picture showing that a tab can be torn out and moved to its own window.](images/tab-new-window.png)
+
+See the [TabView tear out sample](https://github.com/windows-toolkit/Sample-TabView-TearOff/tree/master/TabViewTear) for a more complete sample.
+
+``` xml
+<TabControl CanDragItems="True"
+            CanReorderItems="True"
+            TabDraggedOutside="TabControl_TabDraggedOutside">
+```
+``` csharp
+// NOTE: The app is responsible for writing this code. We will provide a sample that may look something like:
```

Comment:
What will happen if this code isn't written? (Or the event handled in general?)
Will the tab just disappear? Or will it simply return to where it was before?

In what scenarios might a different behavior than opening in a new window be wanted? Are there considerations that need to be made for these other scenarios?

Will a sample also be included that shows how to handle tearing off a tab that was created from an tab in a data bound collection?

Reply:
> What will happen if this code isn't written? 

The tab will simply return to where it was before. 

> In what scenarios might a different behavior than opening in a new window be wanted?

We expect that most apps will want to spawn new windows when tearing out; the motivation for this code to be handled by the app and not the control itself is to maintain a separation of concerns. The UWP windowing story is nuanced (with multi-threaded CoreApplicationView) and evolving (with AppWindow), so rather than coupling the control implementation with the windowing implementation, we'll provide samples for apps to use as a starting point.

> Will a sample also be included that shows how to handle tearing off a tab that was created from an tab in a data bound collection?

Yes, if this would be helpful and worthwhile.